globals.c

```diff
@@ -0,0 +1,18 @@
+#include "globals.h"
+
+int is_not_genuine_scan_tool;
```

Comment:
All the logic for is_not_genuine_scan_tool should be entirely removed as a separate commit.